.github/workflows/scripts/build-linux.sh

```diff
@@ -99,16 +125,65 @@ compile() {
 	export CCACHE_COMPRESSLEVEL="6"
 	export CCACHE_MAXSIZE="400M"
 
-	cd build
-	# Prepare the Cache
+	# # Prepare the Cache
 	ccache -p
 	ccache -z
 	# Build
-	make -j10 install
+	make -j2
+	make install
 	# Save the Cache
 	ccache -s
 }
 
+appimage() {
+	# Create AppDir, and copy over necessary contents from ./bin
+	# Assumes being ran from root of the repository
+	rm -R ./.appdir | true
+	mkdir -p ./.appdir/usr/bin/
+	cp -R ./bin/docs ./.appdir/usr/bin/
+	cp -R ./bin/Langs ./.appdir/usr/bin/
+	cp -R ./bin/shaders ./.appdir/usr/bin/
+	# Assumes plugins have been built
+	cp -R ./bin/plugins ./.appdir/usr/bin/
+	cp ./bin/cheats_ws.zip ./.appdir/usr/bin/
+	cp ./bin/GameIndex.dbf ./.appdir/usr/bin/
+	# NOTE - we cannot distribute a portable AppImage because then it tries to
+	# write to it's read-only file-system
```

Comment:
:wave: According to the [documentation](https://github.com/AppImage/AppImageKit#special-directories) for special directories, it may be possible to provide these as an external directory allowing for portability.

Maybe having something like `PCSX2.config` could be used to store the ini file? Granted this requires some prior knowledge to ensure those files are created.

Also this potentially allows to have the plugins side loaded in a similar manner, thus allowing for custom plugins to be loaded (Unsure if this is currently an option).

Reply:
👋 That's good to know, sounds like we can make an AppImage with portable mode but might take some doing.  There's a lot in flux that might make this difficult (you mentioned plugins, they are currently being incorporated into the core as we speak).  But also the configuration setup in general may change.

However the big unknown to me is how pcsx2 determines the folder paths, I assume it just uses the binary's location but I'm not 100% sure.  If it does, there will likely need to be a code change to allow the `portable.ini` file specify a location.  Or maybe the AppImage documentation you reference is enough (needs to be tested).

Reply:
hi @xTVaser, the AppImage uses the AppDir as root </> and then all folders are relative to this. 
Like in a Linux FileSystem /usr/bin would be AppDir/usr/bin. 
By default on first run, PCSX2 will write to the users actual ~/.config/PCSX2. Those files will be read/write.
AppImages respect creating a custom user home or user config by creating a folder with the same name and append .home or .config to the folder.
`PCSX2.AppImage` would be the app, and if found in the same directory `PCSX2.AppImage.config` all the writeable folders/files will go there. This is how I create a portable PCSX2. 

AppImages can also make use of an updater, so I have PCSX2 update when I pull new images (weekly)

Also, it's bad practice to zip or tar the AppImage. [AppImage Packaging Guide](https://docs.appimage.org/packaging-guide/distribution.html#do-not-put-appimages-into-other-archives)